src/totp-auth.md

```diff
@@ -74,6 +74,14 @@ to the "Details" header. These values can then be entered manually:
 Enrollment is completed once the user enters a valid authentication code
 generated by their device using the provided key.
 
+:::{important}
+If the user does not confirm/complete the enrollment process, the next time the
+users logs in they will be asked to go through the enrollment process, again, and
```

Comment:
Typo: "users"? Sounds like this is talking about a single user.

Reply:
Fixed via rebase.